h2o-parsers/h2o-parquet-parser/src/main/java/water/parser/parquet/ParquetInt96TimestampConverter.java

```diff
@@ -50,8 +48,8 @@ public static long getTimestampMillis(Binary timestampBinary) {
     byte[] bytes = timestampBinary.getBytes();
 
     // little endian encoding - bytes are red in inverted order
-    long timeOfDayNanos = Longs.fromBytes(bytes[7], bytes[6], bytes[5], bytes[4], bytes[3], bytes[2], bytes[1], bytes[0]);
```

Comment:
We can also use UnsafeUtils.get8/get4 to do the same.

Reply:
These functions are the ones used in Guava for exact same functionality